fieldtracing/fieldtracing.cpp

```diff
@@ -1252,46 +1247,47 @@ namespace FieldTracing {
             // Handle flux ropes
             mpiGrid[id]->parameters[CellParams::FLUXROPE] = 0;
             const std::array<Real, 3> x = mpiGrid.get_center(id);
-            // Earlier, if we marked nothing (e.g. hit a wall or ionosphere before making a call) reducedCellXWConnection[n] is less than N_TYPES.
+            // We use cellXWConnection as we swapped with reducedCellXWConnection.
+            // Earlier, if we marked nothing (e.g. hit a wall or ionosphere before making a call) cellXWConnection[n] is less than N_TYPES.
```

Comment:
I'll try to look in more detail at this soon, but I think one needs to be careful now with doing swaps and then going back and forth. :)